### PR TITLE
docs: add adopted stylesheets workaround to FAQ

### PR DESCRIPTION
**Pull request by Westbrook**

### CODE REVIEW

**Comment by thepassle** [comment]
Text LGTM, looks like prettier isnt happy though

**Comment by Westbrook** [comment]
Yeah. Wrote it on GitHub as I’m traveling a bit. Will clean up later.

File: docs/faq/README.md [hunk 1]
```diff
@@ -12,10 +12,75 @@ In this section you can find answers to frequently asked questions regarding jav
...
+```
+
 ## Custom elements render life cycle
 
 <iframe src="https://andyogo.github.io/custom-element-reactions-diagram/" style="width: 940px; border: none;margin-left: -100px; height: 1350px;"></iframe>
 
+## Debugging styles in adopted stylesheets with Chrome Dev Tools
+
+If your custom element leverages the performance benefits of [constructable stylesheets](https://developers.google.com/web/updates/2019/02/constructable-stylesheets), you will likely run into [this Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=946975) that makes it difficult to debug those styles in Dev Tools. In order to make those styles accessible in Dev Tools again, you can prepend the following snippet to the beginning of your development environment to remove the `adoptedStyleSheets` API and have access to those styles in the inspector. When you find yourself needing this work around, it would be beneficial to the community for you to visit the linked Chrome bug and star it to help drive the priority of a fix in the Chrome team.
+
```
Comment (LarsDenBakker) [inline docs/faq/README.md]: Perhaps we can mention here that lit-element uses adopted stylesheets?

Comment (Westbrook) [reply docs/faq/README.md]: Added.

**Comment by Westbrook** [comment]
Added some reference to what actually happens when using this snipper as per conversations on Slack. Having written this now while being in conversation around recent updates to `demoing-storybook`, I wonder if we should make it a default injection or option in our build there... 🤔 but, a conversation for another day.

**Pull request merged**